exalearn-ui/src/app/app.module.ts

```diff
@@ -5,10 +5,33 @@ import { MainPageModule } from './pages/main-page/main-page.module';
 import { AppRoutingModule } from './modules/app-routing/app-routing.module';
 import { AppComponent } from './app.component';
 import { LoginPageModule } from './pages/login-page/login-page.module';
+import { MissingTranslationHandler, TranslateLoader, TranslateModule } from '@ngx-translate/core';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+import { MissingTranslationService } from './services/missing-translation.service';
+
+export function HttpLoaderFactory(http: HttpClient): TranslateLoader {
+	return new TranslateHttpLoader(http, './assets/locale/', '.json');
+}
 
 @NgModule({
 	declarations: [AppComponent],
-	imports: [BrowserModule, AppRoutingModule, MainPageModule, LoginPageModule],
+	imports: [
+		BrowserModule,
+		AppRoutingModule,
+		HttpClientModule,
+		TranslateModule.forRoot({
+			loader: {
```

Comment:
Maybe create above const object for translate module and then put it inside forRoot(..)